build/yamls/base/controller-rbac.yml

```diff
@@ -111,9 +98,110 @@ rules:
       - crdmutator.antrea.tanzu.vmware.com
       - crdvalidator.antrea.tanzu.vmware.com
       - labelsmutator.antrea.io
+      - crdmutator.antrea.io
+      - crdvalidator.antrea.io
```

Comment:
@hongliangl @tnqn Just to confirm: we have to keep the old mutator / validator in addition to the new one? So if a legacy CR is created, and is mirrored, it will go through the webhooks twice?

BTW, the validation code / mutation code only use the new API types. I assume it works on the old legacy CRs because the type definitions are the same?

Reply:
> Just to confirm: we have to keep the old mutator / validator in addition to the new one? So if a legacy CR is created, and is mirrored, it will go through the webhooks twice?
If a legacy CR is created, the legacy one goes through the legacy webhook, and the mirrored new goes through the new webhook. 

> BTW, the validation code / mutation code only use the new API types. I assume it works on the old legacy CRs because the type definitions are the same?
I didn't add anything new in the validation code / mutation code. What I added is `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` in `build/yamls/base/controller.yml`. I have tried using `crdmutator.antrea.io` and `crdvalidator.antrea.io` to mutate/validate the legacy CRs, but it didn't work.